apps/ary/stats.py

```diff
@@ -123,8 +124,8 @@ def get_project_ary_entry_stats(project):
     methodology_attributes_fields = {
         'sampling_size_field_pk': 3,
         # Both are key of methodology field(unit_of_analysis)'s options
-        'households': 11,
-        'individuals': 12,
+        'households': 4,
+        'individuals': 5,
```

Comment:
Are these values supposed to be hard-coded?

Reply:
Yes dai